qutebrowser/browser/webengine/interceptor.py

```diff
@@ -33,6 +33,50 @@ class RequestInterceptor(QWebEngineUrlRequestInterceptor):
 
     """Handle ad blocking and custom headers."""
 
+    # A simple mapping from QWebEngine Resource Types to qutebrowser extension
+    # ResourceTypes. If a ResourceType is added to Qt, this table should be
+    # updated too.
+    QWebEngineResourceTypeMap = {
```

Comment:
This kind of naming is usually used for classes - I'd prefer `RESOURCE_TYPES` or so instead (as it's a constant, and "map" doesn't really say much more).